schema/src/main/scala/CpgExtSchema.scala

```diff
@@ -149,6 +150,9 @@ class CpgExtSchema(builder: SchemaBuilder, cpgSchema: CpgSchema) {
   module.addOutEdge(edge = sourceFile, inNode = file)
   dependency.addOutEdge(edge = sourceFile, inNode = file)
   templateDOM.addOutEdge(edge = sourceFile, inNode = file)
+
+  callNode.addOutEdge(edge = originalProperty, inNode = property)
```

Comment:
Similar to above comment you can replace the different `originalProperty` edges for callNode, member, literal etc with `astNode`